boards/shields/st_b_lcd40_dsi1_mb1166/st_b_lcd40_dsi1_mb1166_a09.overlay

```diff
@@ -4,7 +4,25 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-&mipi_dsi {
+#include <zephyr/dt-bindings/display/panel.h>
+
+/ {
+	lvgl_pointer {
+		compatible = "zephyr,lvgl-pointer-input";
+		input = <&ft5336>;
```

Comment:
If we are going to reference the `ft5336` definition in the shield overlay (which we should, IMO), then we need to define the `ft5336` here as well. We need to define two generic nodelabels within the `stm32h747i_disco` board, for the following:
- some form of nodelabel for the I2C bus the FT5336 IC is present on
- some form of nodelable for the `gpio-nexus` node that describes the connector the FT5336 uses. This is needed so we have a generic way to refer to the pins used by the FT5336 IC across boards.

For an example of board, I'd look at the NXP RT1060 board: 

FT5336 connector definition:
https://github.com/zephyrproject-rtos/zephyr/blob/ca48767be48ca6a7536edf7b1d59f3246046d40e/boards/nxp/mimxrt1060_evk/mimxrt1060_evk.dts#L58

I2C bus definition:
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nxp/mimxrt1060_evk/mimxrt1060_evk.dts#L133